### DISCUSSION TITLE
Ikea Remote E2002 does not report battery level

### DISCUSSION THREAD

**Discussion by tsgwiro1** [Koenkk/zigbee2mqtt#13024] [Devices]
Hi all
I've just installed zigbee2mqtt on my home assistant. I'm on 1.26.0 and home assistant 2022.6.7. As coordiantor I use the Tube's Zigbee Gateways (CC2652P2 variant). The mqtt server (mosquitto) is on a different raspi. The setup works and I connected a Ikea light bulb (LED1924G9) to it with no problems . Afterwards I tried to connect a Ikea Remote E2002. The remote connects but the battery level is not transmitted. I tried to connect again, in fact tried several times but no luck. Tried to connect near the coordiantor and near the bulb, both with no luck. On some of the attempts the interview process even failed... 

The remote does publish the lqi periodically, but the battery is always "null"

Do you have any suggestions on what to do to get the battery level form the remote?

**Comment by coriumalpha** [comment]
Same issue here, if I remember correctly, the first time that I connected it to zigbee2mqtt the battery status was being correctly reported, but after some configuration updates (or unlink/link procedures) now the battery isn't available.

**Reply by coriumalpha** [reply]
I just accidentally solved the issue:
1. I updated the zigbee2mqtt docker image to :latest-dev
2. Paired and unpaired the remote a couple times (left it paired)
3. Went to OTA menu and clicked the Check for updates button (there was no updates available) V 1.0.024
4. With the device paired and working, I removed the battery lid and take out one of the two AAA batteries
5. Then I reinserted the battery and imnediately (and accidentally) pushed one of the buttons
6. Then magically the battery indicator is showing 100%

I don't know if this can help you, but at least I can describe the steps I followed, hope it's useful to you

**Comment by jnswb** [comment]
Found this old discussion when searching solution to same problem. Idea about taking out batteries made me think that Ikea recommends rechargeable batteries (Ni-MH). Their voltage is about 1.2 V compared to normal single use AAA batteries 1.5 V. If the current firmware is optimized for this lower voltage then 1.5 V batteries could give invalid battery charge value.

Anyway, I switched my 1.5 V batteries for 1.2 V Ni-MH batteries and battery level was immediately reported.

**Comment by mgoncaro89** [comment]
This is my first contribution, but I hope it's helpful... I'll detail what I did step by step:

- I opened the battery compartment and removed one of the batteries.
- I put it back in and then held down one of the buttons.
- After that, I went to my Home Assistant, to my Zigbee2MQTT settings, and in the controller options, I pressed the "Reconfigure" button and held down any button on the controller for about 8 seconds.

MAGIC! The battery percentage appeared!
